components/webfield/ProgramChairConsole/PaperStatusMenuBar.js

```diff
@@ -72,9 +72,46 @@ const PaperStatusMenuBar = ({
         }),
         {}
       )),
-    ...(typeof extraPropertiesAllowed === 'object' && extraPropertiesAllowed),
+    ...(typeof extraPropertiesAllowed === 'object' &&
+      Object.fromEntries(
+        Object.entries(extraPropertiesAllowed).map(([key, value]) => {
+          if (typeof value === 'string') {
+            return [key, [key]]
```

Comment:
Can you explain what this is doing? Why is it discarding value if it's a string?